src/app.jsx

```diff
@@ -114,4 +114,4 @@ App.defaultProps = {
   direction: 'bottom',
 };
 
-export default App;
+export default { App, Calendar };
```

Comment:
Make this not such a breaking change. 
I would keep export default app but name export Calendar